admin/views/paper-collapsible.php

```diff
@@ -19,7 +19,7 @@
 	exit();
 }
 ?>
-<div class="paper tab-block" <?php if ( $paper_id ) { echo 'id="' . esc_attr( 'wpseo-' . $paper_id ) . '""'; } ?>>
+<div class="paper tab-block" <?php echo ( $paper_id ) ? 'id="' . esc_attr( 'wpseo-' . $paper_id ) . '""' : ''; ?>>
```

Comment:
I noticed before, there is typo:
```suggestion
<div class="paper tab-block" <?php echo ( $paper_id ) ? 'id="' . esc_attr( 'wpseo-' . $paper_id ) . '"' : ''; ?>>
```

Reply:
While it doesn't really belong with this PR, I totally agree that needs to be fixed, so I've fixed up the commit to include that change.